examples/tv-app/android/java/MessagesManager.cpp

```diff
@@ -0,0 +1,452 @@
+            if (jmessageText != nullptr)
+            {
+                message.messageText = messageText.charSpan();
+            }
+
+            jfieldID messageControlField = env->GetFieldID(messageClass, "messageControl", "I");
+            jint jmessageControl         = env->GetIntField(messageObject, messageControlField);
+            message.messageControl       = static_cast<chip::BitMask<MessageControlBitmap>>(static_cast<uint8_t>(jmessageControl));
+
+            jfieldID priorityField = env->GetFieldID(messageClass, "priority", "I");
+            jint jpriority         = env->GetIntField(messageObject, priorityField);
+            if (jpriority >= 0)
+            {
+                message.priority = MessagePriorityEnum(static_cast<uint8_t>(jpriority));
+            }
+
+            jfieldID startTimeField = env->GetFieldID(messageClass, "startTime", "J");
+            jlong jstartTime        = env->GetLongField(messageObject, startTimeField);
+            if (jstartTime >= 0)
+            {
+                message.startTime = DataModel::Nullable<uint32_t>(static_cast<uint32_t>(jstartTime));
+            }
+
+            jfieldID durationField = env->GetFieldID(messageClass, "duration", "I");
+            jint jduration         = env->GetIntField(messageObject, durationField);
+            if (jduration >= 0)
+            {
+                message.duration = DataModel::Nullable<uint16_t>(static_cast<uint16_t>(jduration));
+            }
+
+            jfieldID getResponseOptionsField =
+                env->GetFieldID(messageClass, "responseOptions", "[Lcom/matter/tv/server/tvapp/MessageResponseOption;");
+
+            jobjectArray responsesArray = static_cast<jobjectArray>(env->GetObjectField(messageObject, getResponseOptionsField));
+            jint size                   = env->GetArrayLength(responsesArray);
+            if (size > 0)
+            {
+                for (jint j = 0; j < size; j++)
+                {
+                    MessageResponseOption option;
+
+                    jobject responseOptionObject = env->GetObjectArrayElement(responsesArray, j);
+                    jclass responseOptionClass   = env->GetObjectClass(responseOptionObject);
+
+                    jfieldID idField         = env->GetFieldID(responseOptionClass, "id", "J");
+                    jlong jid                = env->GetLongField(responseOptionObject, idField);
+                    option.messageResponseID = Optional<uint32_t>(static_cast<uint32_t>(jid));
+
+                    jfieldID getLabelField = env->GetFieldID(responseOptionClass, "label", "Ljava/lang/String;");
+                    jstring jlabelText     = static_cast<jstring>(env->GetObjectField(responseOptionObject, getLabelField));
+                    VerifyOrReturnValue(jlabelText != nullptr, CHIP_ERROR_INVALID_ARGUMENT, ChipLogError(Zcl, "jlabelText null"));
+                    JniUtfString * label = new JniUtfString(env, jlabelText);
```

Comment:
please check whether label is null or not, thanks

Reply:
addressed